### ISSUE TITLE
400 - Invalid redirect_uri

### ISSUE THREAD

**Issue by iEzzat189** [petrbroz/bim360-issue-editor#28]
Basically thank you for this amazing tool.

I'm getting the below error since long time when I try to sign in. However, I'm the account admin of the hub. could you please advise.

![image](https://user-images.githubusercontent.com/108495605/215752619-c4e2585e-4fa2-4725-99a3-6e91969e96bb.png)

**Comment by iEzzat189** [comment]
Hi,

Now I'm getting a different message as in the below snapshot, However I have around 60 project under that account.

![image](https://user-images.githubusercontent.com/108495605/216938653-3b26847c-d198-4d1a-8f43-7714286742ae.png)

**Comment by petrbroz** [comment]
Hi @iEzzat189,

I apologize for the troubles. I've added the correct callback URL to the demo app, so login should now work as expected.

To your second comment: if you want to see your projects in this demo app, you'll need to register it as "custom integration" in your account: https://knowledge.autodesk.com/support/bim-360/learn-explore/caas/CloudHelp/cloudhelp/ENU/BIM360D-Administration/files/About-Account-Admin/GUID-0C83B441-C611-4574-8DA0-45D5CFC235FA-html.html.

**Comment by iEzzat189** [comment]
Hi Petrbroz,

I appreciate your support. It is now working.

My second issue was caused by the change in the Forge Client ID. I was using the old one in the first snapshot. However, when I changed it to the updated one in the second snapshot, it worked fine. I believe that updating the ID in the documentation would be very helpful.

![image](https://user-images.githubusercontent.com/108495605/217191598-39e3663c-9f8a-4eba-b6dc-34a70dc09002.png)

![image](https://user-images.githubusercontent.com/108495605/217191484-a286d47e-9c79-423a-82c1-8c892f05f14c.png)
